tests/conftest.py

```diff
@@ -2950,3 +2951,9 @@ def golden_images_fedora_data_source(golden_images_namespace):
         client=golden_images_namespace.client,
         ensure_exists=True,
     )
+
+
+@pytest.fixture()
+def default_storage_profile_minimum_supported_pvc_size(default_sc):
+    storage_profile = StorageProfile(name=default_sc.name)
+    return storage_profile.instance.metadata.annotations.get("cdi.kubevirt.io/minimumSupportedPvcSize", "6Gi")
```

Comment:
the code is duplicate of `get_default_storage_profile_minimum_supported_pvc_size` function